web/apps/atelier-des-idees/src/components/Modal/PublishIdeaForm/SecondForm/index.jsx

```diff
@@ -0,0 +1,214 @@
+        this.handleErrors = this.handleErrors.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+        this.checkIfComittee = this.checkIfComittee.bind(this);
+    }
+
+    handleErrors() {
+        let canSubmit = true;
+        const verifErrors = Object.keys(this.state.inputs).reduce((acc, curr) => {
+            const isRequired = Object.keys(this.state.errors).includes(curr);
+            const isArrayEmpty =
+				Array.isArray(this.state.inputs[curr]) &&
+				!this.state.inputs[curr].length;
+            if (isRequired && (!this.state.inputs[curr] || isArrayEmpty)) {
+                // TODO: modify error msg
+                acc[curr] = 'Information manquante';
+                if (!acc.form) {
+                    acc.form = 'Certaines informations sont manquantes ou erronées';
+                }
+                canSubmit = false;
+            } else {
+                acc[curr] = '';
+                acc.form = '';
+                canSubmit = true;
+            }
+            return acc;
+        }, {});
+        this.setState({ errors: verifErrors });
+        return canSubmit;
+    }
+
+    handleSubmit(event) {
+        event.preventDefault();
+        if (this.handleErrors()) this.props.onSubmit(this.state.inputs);
+    }
+
+    handleChange(input, value) {
+        this.setState(
+            prevState => ({
+                ...prevState,
+                inputs: { ...prevState.inputs, [input]: value },
+            }),
+            () => this.checkIfComittee(value)
+        );
+    }
+
+    checkIfComittee(value) {
+        if ('committee' === value[0]) {
+            this.setState(prevState => ({
+                ...prevState,
+                inputs: { ...prevState.inputs, committee: [] },
+                errors: { ...prevState.errors, committee: '' },
+            }));
+        } else if ('alone' === value[0]) {
+            this.setState({
+                errors: (({ author, legal }) => ({
+                    author,
+                    legal,
+                }))(this.state.errors),
+                inputs: (({ author, legal, difficulties }) => ({
+                    author,
+                    legal,
+                    difficulties,
+                }))(this.state.inputs),
+            });
+        }
+    }
+
+    render() {
+        return (
+            <form class="second-form" onSubmit={this.handleSubmit}>
+                <div class="second-form__section">
+                    <label class="second-form__section__label">
+						Avec qui avez-vous rédigé cette note ?
+                    </label>
+                    <Select
+                        options={this.props.authorOptions}
+                        placeholder="Seul / Mon comité"
+                        error={this.state.errors.author}
+                        onSelected={value => this.handleChange('author', value)}
+                    />
+                </div>
+                {this.state.inputs.author &&
```

Comment:
Test overkill pouvant masquer une erreur : `this.state.inputs.author` devrait toujours être défini